tests/integration/llm/client.py

```diff
@@ -1356,6 +1375,86 @@ def test_handler(model, model_spec):
                 awscurl_run(req, spec.get("tokenizer"), batch_size)
 
 
+def log_awscurl_benchmark(metric_name: str,
+                          benchmark_name: str = "benchmark.json") -> None:
+    with open(benchmark_name, "r") as f:
+        raw_metrics = json.load(f)
+        metrics = list()
+        metrics.append({
+            "MetricName": f"{metric_name}_p50Latency",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p50Latency"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p90Latency",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p90Latency"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p50TimeToFirstByte",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p50TimeToFirstByte"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p90TimeToFirstByte",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p90TimeToFirstByte"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tokenThroughput",
+            "Unit": "Count/Second",
+            "Value": raw_metrics["tokenThroughput"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tps",
+            "Unit": "Count/Second",
+            "Value": raw_metrics["tps"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tokenPerRequest",
+            "Unit": "Count",
+            "Value": raw_metrics["tokenPerRequest"]
+        })
+        LOGGER.info(f"{metric_name}")
+        LOGGER.info(f"raw metrics: {raw_metrics}")
+        command = f'aws cloudwatch put-metric-data --namespace "serving_handler" ' \
+                  f'--region "us-east-1" --metric-data \'{json.dumps(metrics)}\''
+        LOGGER.info(command)
+        sp.call(command, shell=True)
```

Comment:
not necessarily for this pr, but I feel like we should be using boto3 to publish to cloudwatch rather than a subprocess call. I think we have this in a few other places, so might be worth refactoring later.

Reply:
I agree - I did just follow the subprocess call from previous uses. If we bring in Boto3 it will mean that we need to manage the dependency for some CI tests, which isn't big but is worth noting before we make the refactor.